python/sanic/server.py

```diff
@@ -17,3 +67,8 @@ async def user_info(request, id):
 @app.route("/user", methods=["POST"])
 async def user(request):
     return text("")
+
+if __name__ == "__main__":
+    workers = int(sys.argv[1])
```

Comment:
`sys.argv[1]` will never be set, I prefer to let `sanic` spread across all cores

Reply:
This is to allow passing in $(nproc) in the command line:

```
python server.py $(nproc)
```

I opted for this over `multiprocessing.cpu_count()` since this might not be reliable depending upon the hardware you are using.